TKGame/Character.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame
+{
+    /*internal*/ class Character
+    {
+        Texture2D texture;
+        Vector2 pos;
+        Vector2 vel;
+        bool jumped;
+
+        // Character constructor
+        public Character(Texture2D newTexture, Vector2 newPos)
```

Comment:
Your going to want to add some summary comments in the future to give in code explanations for your functions and variables